service/controller/v1/resource/app/resource.go

```diff
@@ -0,0 +1,80 @@
+package app
```

Comment:
This resource should be called `chart` not `app`. This is because app-operator will watch app CRs and create chart CRs in the tenant cluster.

Reply:
Changed 👍